conf.js

```diff
@@ -7,23 +7,22 @@ exports.config = {
     jasmineNodeOpts: {
         defaultTimeoutInterval: 90000
       },
-    specs: ['./spec/loginSpec.js','./spec/createAccountSpec.js'],
+    specs: ['./spec/loginSpec.js','./spec/createAccountSpec.js','./spec/createAccountNegSpec.js'],
     onPrepare: function(){
         console.log('Start of tests execution');
-        browser.waitForAngularEnabled(false);
         browser.manage().timeouts().implicitlyWait(4000);
```

Comment:
Move this into `beforeEach()` also

Reply:
Moved

Reply:
`browser.manage().timeouts().implicitlyWait(4000);` Move this into beforeEach()